CMakeLists.txt

```diff
@@ -93,7 +93,7 @@ if(APPLE)
 endif()
 
 # Set C++17 as default required standard for all C++ targets.
-set(CMAKE_CXX_STANDARD 17 CACHE STRING "C++ Standard")
+set(CMAKE_CXX_STANDARD 20 CACHE STRING "C++ Standard")
```

Comment:
I know we're planning on migrating to C++20 sooner rather than later, but is this the PR for checking that change in? I'm wondering if this should just be a bootstrap/cmake option to make it build time configurable.

Reply:
This should be left at C++17 for now and this should not be in the PR.  Although cmake had this, VS actually used C++17 for all of the changes I made.